.snyk

```diff
@@ -66,19 +66,19 @@ ignore:
     - '*':
         reason: >-
            Not affecting us since no debugger is enabled in cloud.gov apps
-        expires: 2024-06-31T16:20:58.017Z
+        expires: 2024-09-30T16:20:58.017Z
   SNYK-PYTHON-CRYPTOGRAPHY-7161587:
     - '*':
         reason: >-
            No remediation available yet. Issue tracked in github:
            https://github.com/GSA/data.gov/issues/4781
-        expires: 2024-06-31T16:20:58.017Z
+        expires: 2024-09-30T16:20:58.017Z
```

Comment:
> https://github.com/GSA/data.gov/issues/4781

issue is closed. can we remove this from the ignore list?

Reply:
Double checked with @FuhuXia.  We should keep it as it is since the exceptions are still present, but we've determined that they do not impact our product, so the issue was closed. @btylerburton

Reply:
Ok, yeah that's what I figured since they are related to the `SSL_free_buffers` check.